smac/facade/smac_facade.py

```diff
@@ -226,6 +229,9 @@ def __init__(self,
                 acquisition_function = LogEI(model=model)
             else:
                 acquisition_function = EI(model=model)
+        else:
+            acquisition_function_kwargs["model"] = model
+            acquisition_function = acquisition_function(**acquisition_function_kwargs)
```

Comment:
@mfeurer does that looks like what you wanted for a better SMAC interface?

Reply:
Yes :+1: 

Three suggestions:
1. Use None as the default argument for the acquisition_function_kwargs
2. It would be great if you could check that there's no model in the acquisition_function_kwargs
3. Maybe use the acquisition function kwargs for LogEI and EI, too?

Reply:
Thanks for the feedback. I did it now for all arguments.

regarding your suggestions:

 1. I prefer empty dicts for the kwargs, since I cannot do `dict.update(None)`
 2. I define now default arguments which are overwritten by user arguments
 3. Done.